src/components/MediaGallery/MediaGallery.tsx

```diff
@@ -44,7 +44,7 @@ const MediaGallery = ({ resources: initialResources, tag }: MediaGalleryProps) =
     handler: (_event: KeyboardEvent) => {
       setSelected([]);
     },
-    element: document.body
+    element: typeof document !== "undefined" ?  document.body : null
```

Comment:
In most cases this will be a ref.current, but in this particular case, we're accessing `document.body`, so we need to ensure that DOM element only runs on the clients-side.